src/status_im/contexts/shell/share/view.cljs

```diff
@@ -50,7 +51,7 @@
          {:size           28
           :blur?          true
           :on-change      #(reset! selected-tab %)
-          :default-active :profile
+          :default-active initial-tab
```

Comment:
Should we be refactoring components to a component without a nested render function when we come across them?

Reply:
I think that's on us to decide on how "safe" the refactors are, but leading up to the 2.30 release I think we should keep unnecessary refactors to a minimum unless there is an important reason why, e.g performance issues. 
imo at this point it's advantageous to do whatever we can do to reduce bugs creeping in
cc @cammellos wdyt? 🤔